csharp/src/Ice/ACM.cs

```diff
@@ -2,146 +2,51 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
+using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
+using System.Threading;
 
 namespace ZeroC.Ice
 {
-    internal sealed class ACMConfig
+    /// <summary>This interface represents a monitor for the Acm (Active Connection Management)</summary>
```

Comment:
I assume you plan to rename the file Acm.cs in a future PR?

Reply:
Done